.gitignore

```diff
@@ -42,3 +42,5 @@ profile.cov
 /dashboard/assets/node_modules
 /dashboard/assets/stats.json
 /dashboard/assets/bundle.js
+/dashboard/assets/yarn-error.log
+/dashboard/assets/package-lock.json
```

Comment:
You want package lock committed to the repo. That's what ensures reproducible builds.

Reply:
Ah, there's yarn.lock. sry